src/VisualStudio/CodeLens/ReferenceCodeLensProvider.cs

```diff
@@ -158,12 +166,12 @@ string GetCodeElementKindsString(CodeElementKinds kind)
                 }
             }
 
-            public async Task<CodeLensDetailsDescriptor> GetDetailsAsync(CancellationToken cancellationToken)
+            public async Task<CodeLensDetailsDescriptor> GetDetailsAsync(CodeLensDescriptorContext descriptorContext, CancellationToken cancellationToken)
             {
                 // we always get data through VS rather than Roslyn OOP directly since we want final data rather than
                 // raw data from Roslyn OOP such as razor find all reference results
-                var referenceLocationDescriptors = await _vsCallbackRpc.InvokeWithCancellationAsync<IEnumerable<ReferenceLocationDescriptor>>(
-                    nameof(ICodeLensContext.FindReferenceLocationsAsync), new object[] { this.Descriptor }, cancellationToken).ConfigureAwait(false);
+                var referenceLocationDescriptors = await _callbackService.InvokeAsync<IEnumerable<ReferenceLocationDescriptor>>(
+                    _owner, nameof(ICodeLensContext.FindReferenceLocationsAsync), new object[] { this.Descriptor, descriptorContext }, cancellationToken).ConfigureAwait(false);
 
                 var details = new CodeLensDetailsDescriptor
                 {
```

Comment:
on line 210 below it should be 
new CodeLensDetailEntryField() { Text = referenceLocationDescriptor.FilePath },
instead of 
new CodeLensDetailEntryField() { Text = Descriptor.FilePath },